config/schema/synonyms.yml

```
@@ -102,6 +102,7 @@
 # Unofficial or deprecated terms should expand to the preferred term
 - search: ancestral visa => ancestry visa
 - search: bedroom tax => bedroom tax, spare bedroom, spare room
+- search: brexit => eu exit
```

Should this be as follows so it includes both terms?

```suggestion
- search: brexit => brexit, eu exit
```

---

Good catch!